src/main/java/de/cerus/jdasc/JDASlashCommands.java

```diff
@@ -214,20 +214,41 @@ private JDASlashCommands() {
      * @return The command id
      */
     public static CompletableFuture<Long> submitGlobalCommand(final ApplicationCommand command, final ApplicationCommandListener listener) {
-        return discordHttpClient.submitGlobalCommand(command).thenApply(response -> {
-            final JsonObject object;
-            try {
-                object = JsonParser.parseString(response.body().string()).getAsJsonObject();
-            } catch (final IOException e) {
-                throw new CompletionException(e);
+        try {
+            List<ApplicationCommand> globalCommands = getGlobalCommands().get();
+            if(globalCommands.contains(command)){
+                return getLongCompletableFuture(command, listener, globalCommands);
```

Comment:
Could be replaced by [CompletableFuture.completedFuture()](https://docs.oracle.com/javase/8/docs/api/java/util/concurrent/CompletableFuture.html#completedFuture-U-)